lib/MySQL_PreparedStatement.cpp

```diff
@@ -672,6 +677,34 @@ void MySQL_STMT_Manager_v14::ref_count_server(uint64_t _stmt_id ,int _v, bool lo
 		pthread_rwlock_unlock(&rwlock_);
 }
 
+void MySQL_STMTs_local_v14::remove_stmt(MYSQL_STMT* stmt, bool fromExecutedState) {
+	if (fromExecutedState)
```

Comment:
As discussed with @val214 , this code and the parameter are no longer necessary because we are not dealing for now with `PROCESSING_STMT_EXECUTE` state related errors.

Reply:
Fixed